source/quick-start.txt

```diff
@@ -37,11 +37,62 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
-
 After you verify that you installed the required versions of Node and NPM,
-create a new project.
+create a new project by :ref:`cloning our js-starter repository
+<qs-download-the-repo>` or by :ref:`creating it manually <qs-manual-setup>`.
+
+.. _qs-download-the-repo:
+
+Clone the Project
+~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   This approach requires you to have `Git
+   <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+   installed.
```

Comment:
```suggestion
Install `git <git installation instruction url>`__
```

Suggestion: I think this should be a required step in the tutorial as it is a requirement.

Idea: Could these steps perhaps be put in a numbered list? I was requested this recently from Melissa in a guides page and I thought it worked well.

Reply:
I'll update this to be the first step but will not implement a numbered list.